<?xml version="1.0" encoding="UTF-8"?>
<!-- Path: gui\bulk_produtos_versoes_historicas\bulk_produtos_versoes_historicas_dialog.ui -->
<ui version="4.0">
 <class>LoadHistoricalProductsDialog</class>
 <widget class="QDialog" name="LoadHistoricalProductsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Produtos com Versões Históricas</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Adicionar Produtos com Versões Históricas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Esta ferramenta permite adicionar produtos com versões históricas (sem arquivos) ao banco de dados. Você deve selecionar uma camada tabular contendo os dados necessários.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Seleção de Camada</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Camada:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="layerComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createModelLayerButton">
        <property name="text">
         <string>Criar Camada Modelo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Requisitos da Camada</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>A camada deve ser uma tabela com os seguintes campos:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Campos do Produto (obrigatórios):
• produto_grupo_id: Identificador para agrupar versões de um mesmo produto
• produto_nome: Nome do produto
• tipo_escala_id: Tipo de escala do produto
• tipo_produto_id: Tipo de produto
• geom: Geometria do produto em formato WKT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Campos do Produto (opcionais):
• mi: MI do produto
• inom: INOM do produto
• denominador_escala_especial: Denominador para escala especial (tipo 5)
• descricao_produto: Descrição do produto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Campos da Versão (obrigatórios):
• versao_grupo_id: Identificador para agrupar versões
• versao: Número da versão
• nome_versao: Nome da versão
• orgao_produtor: Órgão produtor da versão
• data_criacao: Data de criação da versão (formato ISO)
• data_edicao: Data de edição da versão (formato ISO)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Campos da Versão (opcionais):
• lote_id: ID do lote associado
• descricao_versao: Descrição da versão
• palavras_chave: Lista separada por vírgulas
• metadado_versao: Metadados em formato JSON</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Selecione uma camada e clique em Carregar para iniciar o processo.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Carregar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Fechar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>LoadHistoricalProductsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>